utils/pkg/pkg_run

```diff
@@ -158,9 +162,22 @@ package_cloud_push () {
       export PC_REPO="worker-testing"
     fi
 
-    package_cloud push \
-      $PC_USER/$PC_REPO/$package_cloud_os/$platform_release \
-      tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype --verbose
+    PC_UPLOAD_FILE=$(ls $CHECKOUT_ROOT/tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype)
```

Comment:
I think you can probably safely swap out `ls` with `echo` and let the shell do the expansion.  You might need/want to do some `shopt -s nullglob` so as not to end up with the unexpanded glob as your only entry.